test/tests/functional/pbs_hook_execjob_prologue.py

```diff
@@ -216,5 +216,26 @@ def test_prologue_hook_set_job_attr(self):
         attr = {'resources_used.file': '2gb'}
         self.server.expect(JOB, attr, id=jid, extend='x', offset=1)
         self.server.accounting_match(
-            "E;%s;.*%s.*" % jid, 'resources_used.file=2gb', regexp=True,
+            "E;" + jid + ";.*resources_used.file=2gb.*", regexp=True,
             max_attempts=10)
+
+    def test_prologue_hook_fail_action_non_mom_hook(self):
+        """
+        test that the fail action is set to anything else then none
+        and the mom hook is not one of the follwing execjob_begin,
+        exechost_startup, execjob_prologue error msg is displayed
+        """
+        hook_name = "prologue"
+        hook_body = ("import pbs\n"
+                     "pbs.event().accept()\n")
+        attr = {'event': 'exechost_periodic',
+                'fail_action': 'offline_vnodes'}
+        try:
+            self.server.create_import_hook(hook_name, attr, hook_body)
+        except PbsManagerError, e:
+            exp_err="qmgr obj=prologue svr=default: " \
+                    "Can't set hook fail_action value to 'offline_vnodes': " \
+                    "hook event must" \
+                    " contain at least one of execjob_begin" \
+                    ", exechost_startup, execjob_prologue"
```

Comment:
New guidelines from the PTL steering committee just came out.  Error messages should be +='d together.